src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -67,6 +68,7 @@
 	private final ConfigurableFileCollection unmappedMods;
 
 	final List<LoomDecompiler> decompilers = new ArrayList<>();
+	final List<JarProcessor> jarProcessors = new ArrayList<>();
```

Comment:
Why package private?

Reply:
I thought of using it in LoomGradlePlugin but that didn't go well